lib/compilers/fortran.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017, Rubén Rincón
```

Comment:
I don't recall ever writting this, hmm

Reply:
Ah, yes. I copied the haskell.js as boiler plate. So I delete this line?

Reply:
You should put your name there.
I think you might be able to just put `Compiler Explorer Authors` if you do not wish to have anything to do with it, but ianal

Reply:
Changed w/ latest commit.